Tests/OpenAPIGeneratorReferenceTests/Resources/ReferenceSources/Petstore/Types.swift

```diff
@@ -154,25 +154,89 @@ extension APIProtocol {
 /// Server URLs defined in the OpenAPI document.
 public enum Servers {
     /// Example Petstore implementation service
+    public enum Server1 {
+        /// Example Petstore implementation service
+        public static func url() throws -> Foundation.URL {
+            try Foundation.URL(
+                validatingOpenAPIServerURL: "https://example.com/api",
+                variables: []
+            )
+        }
+    }
+    /// Example Petstore implementation service
+    @available(*, deprecated, renamed: "Servers.Server1.url")
```

Comment:
```suggestion
    @available(*, deprecated, renamed: "Servers.Server1.url()")
```

Should it also contain the `()` or not? Can you check that accepting the Fix-it does the right thing?

Reply:
I tested this as I was writing it, and it seems to not matter. Whether the renamed function takes no parameters, many parameters, parameters with defaults the Fix-It just seems to work; weirdly I was even able to get Fix-Its like `"Servers.Server3.url(a:)"` to work as well.

Reply:
Fair enough, thanks for checking. Let's leave it as-is.